compiler/packages/babel-plugin-react-compiler/src/__tests__/fixtures/compiler/ref-like-name-as-refs-3.js

```diff
@@ -0,0 +1,22 @@
+// @enableTreatRefLikeIdentifiersAsRefs @validatePreserveExistingMemoizationGuarantees
+import { useRef, useEffect } from "react";
+
+function useCustomRef() {
+  return useRef({ click: () => {} });
+}
+
+function Foo() {
+  const ref = useCustomRef();
+
+  useEffect(() => {
+    ref.current?.click();
+  }, []);
```

Comment:
ah so this is useEffect and the one below is in a callback function. small nit, but maybe name like `ref-like-name-effect`, `ref-like-name-callback` etc

Reply:
Done